tests/fsharpqa/Source/CompilerOptions/fsc/flaterrors/env.lst

```diff
@@ -10,10 +10,10 @@ ReqENU	SOURCE=E_MultiLine01.fs								# E_MultiLine01.fs
 # Last one wins... (multiple-usage)
 	SOURCE=E_MultiLine02.fs COMPILE_ONLY=1 SCFLAGS="--flaterrors --flaterrors"	# MultipleUse
 
-# Option is case sentitive
+# Option is case-sensitive
 	SOURCE=E_MultiLine04.fs COMPILE_ONLY=1 SCFLAGS="--FlatErrors"			# CaseSensitive01
 	SOURCE=E_MultiLine04.fs COMPILE_ONLY=1 SCFLAGS="--FLATERRORS"			# CaseSensitive02
 
-# Mispelled options
-	SOURCE=E_MultiLine04.fs COMPILE_ONLY=1 SCFLAGS="-flaterrors"			# Mispelled01
-	SOURCE=E_MultiLine04.fs COMPILE_ONLY=1 SCFLAGS="--flaterrors+"			# Mispelled02
\ No newline at end of file
+# Misspelled options
```

Comment:
Hah, wonder if this was meant to be a hidden joke here

Reply:
Quite possibly, but sadly it came with the original source release so it's very hidden